### ISSUE TITLE
Comments on the docsite

### ISSUE THREAD

**Issue by TomDonoghue** [mne-tools/mne-icalabel#86]
Collecting a few small / quick comments on the documentation website. 
This comment / review is part of the JOSS paper review: 
https://github.com/openjournals/joss-reviews/issues/4484

General:
- In the `About` part of the repo page (top right), it would be useful to put the documentation site in the link field
- There seems to be small issue with the version selector (multiple "v0.1"s)
    - ![Screen Shot 2022-07-31 at 2 23 57 PM](https://user-images.githubusercontent.com/7727566/182040099-c78e4be5-0e55-465f-a2ee-528fef4ea1d6.png)
- The License button in the header is a broken link
- I think the landing page of the docsite should say a bit more about this project, including a bit more description (such as is in the README / summary of the JOSS paper), and a note that this is a re-implementation of the ICLabel project. 

Example:
- There are some parts that are not rendering correctly (as in, what should be text markdown getting rendered as a code block)
    - This includes the section "# Example: EOG and ECG artifact repair", for both 0.2 release, and the dev version
- If installing `mne-icalabel`, with it's required dependencies, and trying to run the example, one hits a dependency error for fitting the ICA (`sklearn`)
    - I think it might be worth mentioning in the example that it has an extra dependency: scikit-learn
- For the cell that plots the ICA sources, this is what I get locally:
    - <img width="696" alt="Screen Shot 2022-07-31 at 2 47 53 PM" src="https://user-images.githubusercontent.com/7727566/182040864-803762f3-4256-46e6-be18-b61826fed16a.png">
    - However, one the docsite, it looks like: 
    - <img width="770" alt="Screen Shot 2022-07-31 at 2 49 22 PM" src="https://user-images.githubusercontent.com/7727566/182040905-2445fbc3-5c6f-40c9-9973-f78c0cf4dc47.png">
    - There appears to be a rendering issue on the docsite, which is an issue for the comments about looking at the time series. 
- After this section of the tutorial, the description and notes discuss "00" as eye activity, and "01" as heartbeat. I agree with "00", but I think something changed at some point such at 01 is not clearly heartbeat, but rather 05 looks more like heartbeat. This is a bit unclear on the docsite, since the time series plot doesn't render, but the time series plot above that gets generated locally seems to suggest this. (Note that the automatic labelling after this also does not label 01 as heartbeat).
- The `Selecting ICA components automatically` again has some rendering issues, and I think should be more clearly split up into separate blocks to see each step here
- There are quite a lot of warnings printed out from the `label_components` function (local version copied here), and this seems to imply there are some issues with what is being computed? If not, why are the warnings arising?
    - <img width="980" alt="Screen Shot 2022-07-31 at 2 57 07 PM" src="https://user-images.githubusercontent.com/7727566/182041177-077fe3e5-6ad5-431a-94a1-a26ed4dd823a.png">
- After the labels have been computed, rather than printing them out as a list that is a disconnected from the actual ICs, I think it would be much more informative to then print out 
    - something like `for ind, label in enumerate(labels): print(label); ica.plot_properties(raw, picks=[ind]);`
    - ^It might be a bit much to print _all_ of them, but for a subset showing "this is an IC that the labelling called X" would be nice.

**Comment by adam2392** [comment]
@agramfort are you able to provide me with admin access to the repo? I can't change the `About` field in the repo

**Comment by agramfort** [comment]
Done

**Comment by mscheltienne** [comment]
Thank you for the detailed feedback! We will have a look at those items.

**Comment by adam2392** [comment]
Thanks for the feedback! Will add a comment for each item.

> In the About part of the repo page (top right), it would be useful to put the documentation site in the link field

Added!

> There seems to be small issue with the version selector (multiple "v0.1"s)
> The License button in the header is a broken link

Fixed both license and drop-down menu: https://mne.tools/mne-icalabel/v0.1/index.html#indices-and-tables

**Comment by adam2392** [comment]
Hi @TomDonoghue I have addressed the remainder of your comments around the example and documentation: https://mne.tools/mne-icalabel/dev/index.html

Thanks so much for the feedback! lmk if you have other issues w/ how we reformatted/re-wrote the example.

**Comment by TomDonoghue** [comment]
This update looks good to me!
